game-core/src/main/java/games/strategy/net/MacFinder.java

```diff
@@ -37,7 +38,8 @@ private MacFinder() {}
   public static String getHashedMacAddress() {
     final String mac = getMacAddress();
     if (mac == null) {
-      throw new IllegalArgumentException("You have an invalid MAC address or TripleA can't find your mac address");
+      throw new IllegalArgumentException("You have an invalid MAC address or TripleA can't" 
+          + " find your mac address");
```

Comment:
@DanVanAtta Is your formatter set to 120 columns?  There seem to be several changes in this PR that simply re-wrap lines without any actual modifications.

Reply:
>  There seem to be several changes in this PR that simply re-wrap lines without any actual modifications.

Yes; it was actually a problem with the long lines; i was not able to see the ends of them without scrolling. I was doing a side by side with code commentary.

> Is your formatter set to 120 columns?

It was actually checkstyle, needed to update it explicitly to tripleA and did not have that right away. That was done. Regardless, wrapping to shorter line lengths is up to our own discretion.

Reply:
@ssoloff should deprecate every method on MD5crypte instead fo doing package imports. That fix one way or another is out of scope here.

We should also probably follow up on warnings. The travis output warnings are the official ones for the project. They are clogged with javadoc and variable naming checkstyle problems. (as an example, tune the eclipse warnings all the way to max, suddenly the deprecation warnings seem a bit less urgent). But also of consequence, if we deprecate a heavily used class and that drowns out other warnings, that is also a problem. It means we cant' deprecate properly as it requires a near immediate fix. Undeprecating a deprecated class just to not get the warnings is not right. Also if the concern is that more important warnings will go unnoticed, then those more important warnings do not have the right severity, or the warnings we are see do not have the right severity and should be suppressed outright.

Reply:
In the Gradle build, compiler warnings are output in a separate task from Checkstyle warnings, so it's easy to distinguish between the two, especially if you only run the `compile*Java` tasks.

I'm not concerned by the noise of Checkstyle warnings (currently ~1,850) because the build enforces that no new Checkstyle warnings are introduced.

I am concerned about introducing new compiler warnings (was ~125 when Error Prone was present) because we don't automatically check these as part of the build.  When I remember to do so, I'll typically just scan the Gradle output of the `compile*Java` tasks to see if there's anything new.  That's much easier to do without a bushel full of `Md5Crypt` deprecation warnings. :smile:

Regardless, I think the plan going forward should be to un-deprecate `Md5Crypt` and qualify what its purpose is (and is not) via Javadocs.  If desirable, we can split the existing `crypt()` method into two methods (e.g. `cryptSensitive()` and `cryptUnsensitive()`) and only deprecate the "sensitive" version.  When we eventually remove MD5-crypted passwords from the lobby, usage of the sensitive version should disappear, and we can go back to a single `crypt()` method.  (Note that I don't expect that to be done in this PR.)

So, please consider my concern about removing the fully-qualified method names withdrawn. :smile:

Reply:
> I  am concerned about introducing new compiler warnings (was ~125 when Error Prone was present) because we don't automatically check these as part of the build. 

Not reporting on these in any way seems to be a gap for us. WE should at least keep track of the trend. Most compiler warnings can be 'tuned up', I typically do this on any IDE environment (the eclipse defaults are pretty nice, but there are many quite useful additional warnings that can be turned on). So; I'm wondering a bit about compiler warnings, if we can up-tune those so we are not using a default set (that would be a bit arbitrary).

>  method into two methods (e.g. cryptSensitive() and cryptUnsensitive()) and only deprecate the "sensitive" version.

I'm in agreement with this

Reply:
@ssoloff a quick response to:
> I'm not concerned by the noise of Checkstyle warnings (currently ~1,850) because the build enforces that no new Checkstyle warnings are introduced.

This problem actually does concern me. I'll often do many commits, splitting out machine refactoring from hand refactoring, to then find out there is a +1 checkstyle error somewhere. Undoing commits so I can do a scan on updated files can be painful, and still painful as most files then have dozens of existing violations. The noise of violations is a problem, it slows down development.